### ISSUE TITLE
icbuild.py doesn't run in Windows

### ISSUE THREAD

**Issue by esodan** [nacho/icbuild#1]
I'm trying to execute from command line in Windows:

python icbuild.py

it fails at line 32 on icbuild module, fails to find jhbuild module to import jhbuild.environment

Where can I get jhbuild module and install in my Windows Box?

**Comment by nacho** [comment]
yes, icbuild is still far from working, any help to get it working would be great. See that it should be run with pyhon3 though

**Comment by nacho** [comment]
see that icbuild is a fork from jhbuild, so you need to read the code an check what needs to be renamed and or fixed.

**Comment by esodan** [comment]
Do you mean icbuild should download and compile modules? Why don't simply
patch jhbuild to be executed on Windows?

Are there a way to create VS projects files to build Gtk and develop
applications against it?

**Comment by pbor** [comment]
What nacho meant is that icbuild is still not ready at all, the idea behind it was to have something like jhbuild for windows, but since these days jhbuild is pretty big and complex, we tried out with "icbuild" taking only small parts and trying to build something more minimalistic.

icbuild is definitely not ready to be used.

If you are just looking for a script to build gtk and use it your app, at the moment yuo should use https://github.com/wingtk/gtk-win32 that's what we use in production for our apps
